css/selectors/focus-visible-013.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>CSS Selectors Test: :focus-visible does not match after mouse focus even if previous focused element was matching :focus-visible</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo" />
+<meta name="assert" content="This test checks that if the active element matches ':focus-visible' and a mouse click causes focus to move elsewhere, the newly focused element should not match ':focus-visible'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<style>
+  @supports not (selector(:focus-visible)) {
+    :focus {
+      outline: red solid 5px;
+    }
+  }
+
+  #initial:focus-visible {
+    outline: green solid 5px;
+  }
+
+  #initial:focus:not(:focus-visible) {
+    outline: red solid 5px;
+  }
+
+  #target:focus-visible {
+    outline: red solid 5px;
+  }
+
+  #target:focus:not(:focus-visible) {
+    outline: green solid 5px;
+  }
+</style>
+
+<div id="initial" tabindex="0">Initial</div>
+<div id="target" tabindex="0">Target</div>
+
```

Comment:
Can we have some manual testing instructions as well?

Reply:
These tests are automatic, so I'm not sure why we need the manual testing instructions. I can add them though.

Reply:
I use the manual testing version when I am debugging browser code during implementation, and I figure others might like to as well.

Reply:
Ok, I'll add them.